editor/src/components/canvas/canvas-strategies/shared-absolute-move-strategy-helpers.ts

```diff
@@ -179,3 +180,21 @@ export function getMultiselectBounds(
 
   return boundingRectangleArray(frames)
 }
+
+export function getFileOfElement(
+  target: ElementPath | null,
+  projectContents: ProjectContentTreeRoot,
+  openFile: string | null | undefined,
+): string | null {
+  let elementFile: string | null = null
+  forUnderlyingTarget(
```

Comment:
`withUnderlyingTarget` might suit this better as it handles the default case for you.

Reply:
Nice, fixed in a30bd405851c6803a3481c7b7a8dc884c6cec297